tests/components/flashforge/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""Tests for the Flashforge integration."""
+
+from typing import Any
+from unittest.mock import Mock
+
+import voluptuous as vol
+
+from homeassistant.components.flashforge.const import CONF_SERIAL_NUMBER, DOMAIN
+from homeassistant.const import CONF_IP_ADDRESS, CONF_PORT
+from homeassistant.core import HomeAssistant
+
+from .const_response import (
+    MACHINE_INFO,
+    PROGRESS_PRINTING,
+    PROGRESS_READY,
+    STATUS_PRINTING,
+    STATUS_READY,
+    TEMP_PRINTING,
+    TEMP_READY,
+)
+
+from tests.common import MockConfigEntry
+
+
+def prepare_mocked_connection(mocked_obj: Mock) -> Mock:
```

Comment:
You could define these as `fixtures` and add them to `conftest.py` instead
(same to other fixtures below).